docs/source/search-runs.rst

```diff
@@ -93,139 +93,183 @@ Then paste the following code to a local pythnon file, e.g., ``~/create_testing_
-concatenated by ``and``. At a high level, the syntax is similar to SQL WHERE clause without supporting ``or`` keyword.
-For example, below is a query to search for all runs that have logged a ``loss`` metric smaller than ``1`` to
-MLflow.
+To filter your MLflow runs, you need to write **search queries**, which are SQL-like conditions. 
 
-.. code-block:: sql
+To showcase this functionality, let's reopen the MLflow UI, let's explore some example fields that you can use to filter your runs in the MLflow UI.
+
+.. figure:: _static/images/search-runs/before_search_query.png
+   :alt: search query
+   :width: 90%
+   :align: center
 
-  metric.loss < 1
+Second, let's try filtering the runs for our really bad models: ``metrics.loss > 0.8``.
 
-Briefly a search condition consists of 3 parts:
+.. figure:: _static/images/search-runs/after_search_query.png
+   :alt: search query
+   :width: 90%
+   :align: center
 
-- An MLflow field on the left side, which can be some metric, param, tag, dataset or MLflow run metadata, e.g.,
-  ``metric.loss`` or ``param.batch_size``.
-- A comparator in the middle, we support 3 types of comparators:
+You'll notice that we now are displaying 2 runs instead of 10. Pretty easy, right?
 
-  - Numeric comparators: ``=``, ``!=``, ``>``, ``>=``, ``<``, and ``<=``.
-  - String comparators: ``=``, ``!=``, ``LIKE`` and ``ILIKE``. String comparators are case sensitive except ``ILIKE``
-    which is case insensitive.
-  - Set comparators: ``IN``, which only supports searching by dataset and run metadata, and is case sensitive.
+Now let's go over the search query syntax in more detail.
 
-- A reference value on the right side.
+Search Query Syntax Deep Dive
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. note::
-  There is a caveat that using ``IN`` requires wrapping string with single quotes, e.g.,
-  ``params."learning rate" IN ('0.001', '0.01')``. ``params."learning_rate" IN ("0.001", "0.01")`` will throw an error.
+As noted above, MLflow search syntax is similar to SQL with a few notable exceptions.
 
-Handling Special Characters in MLflow Field
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* The SQL ``OR`` keyword is not supported.
+* For fields that contain special characters or start with numbers, you need to wrap them in double quotes.
 
-It's common that your MLflow field contains special characters such as hyphen, space, period, and so on. In those
-cases, you will need to wrap it by double quotes. The same rule applies when your field starts with numbers.
-For example if you ever logged something by ``mlflow.log_metric("cross-entropy-loss", 0.15)``, then your query will be
-similar to:
+  * Good: ``metrics."cross-entropy-loss" < 0.5``
+  * Bad:  ``metrics.cross-entropy-loss < 0.5``
+  * Good: ``params."1st_iteration_timestamp" = "2022-01-01"``
+  * Bad:  ``params.1st_iteration_timestamp = "2022-01-01"``
 
-.. code-block:: sql
+* For the SQL ``IN`` keyword, you must surround the values of your list with single quotes.
```

Comment:
For the good / bad examples, can we make it more visually apparent the difference between them?

```rst
.. code-block:: diff

    - Bad: params."learning rate" IN ("0.001", "0.01")
    + Good: params."learning rate" IN ('0.001', '0.01')
```
(might require some adjustment, but is likely to be very obvious at a quick glance which one to pay attention to for a quick reference)